kubos/test/integration/integration_test.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+
+import json
+import kubos
+import logging
+import mock
+import os
+import sys
+import unittest
+
+from sets import Set
+from kubos.test.utils import KubosTestCase
+from kubos.utils.constants import GLOBAL_TARGET_PATH
+
+class CLIIntegrationTest(KubosTestCase):
+
+    def setUp(self):
+        super(CLIIntegrationTest, self).setUp()
+        self.proj_name = 'project-test'
+        self.proj_dir = os.path.join(self.base_dir, self.proj_name)
+        self.target_list = None #Defined at test runtime
+        self.first_arg = sys.argv[0]
+        sys.argv = [self.first_arg] #clear any excess command line arguhements
+        logging.error = mock.MagicMock()
+
+
+        #The tests are run in alphabetical order by default (hence the 1,2,3,etc..)
+        #TODO: Get a better way of running the test in order while still being readable
+    def test_1_update_source_check_versions(self):
+        self.run_command('update')
```

Comment:
@plauche: Any thoughts on this and the changes I need to make to get this wrapped up?